dockers/docker-orchagent/switch.json.j2

```diff
@@ -26,8 +26,10 @@
             "ecmp_hash_seed": "{{ hash_seed_value }}",
             "lag_hash_seed": "{{ hash_seed_value }}",
             "fdb_aging_time": "600",
+{% if not DEVICE_METADATA.localhost.switch_type or DEVICE_METADATA.localhost.switch_type != "chassis-packet" %}
```

Comment:
@anamehra : change this condition to be more platform specific where if asic_type is `cisco-8000` and topology is  `t2` we don;t set hash offset explicitly